services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -289,8 +296,7 @@ export class NoticeOfWorkApplication extends Component {
       .then(() => finalAction());
   };
 
-  openUpdateLeadInspectorModal = (event) => {
-    event.preventDefault();
+  openUpdateLeadInspectorModal = () => {
```

Comment:
I restructured things so this isn't needed here anymore and for similar use-cases.